neps/nep-0616.md

```diff
@@ -313,6 +313,17 @@ sequenceDiagram
     deactivate C
 ```
 
+It also makes sense to let smart-contracts know in the runtime which refund
+account was set for the current receipt by its predecessor via `promise_set_refund_to()`,
+so that they can use it according to its business logic instead of duplicating
+it in a FunctionCall arguments. So, we might need another host function for that:
+
+```rust
+/// Returns the [`AccountId`] that was set for the current receipt by its
+/// predecessor via [`set_refund_to()`], or [`predecessor_account_id()`] otherwise.
+pub fn get_refund_to() -> AccountId;
+```
+
```

Comment:
Makes sense to me!

Just one bikeshedding: No existing host function uses the `get_*`prefix. I think a more consistent name would be `refund_to_account_id`.

Reply:
Oh and the host function for setting this is named `promise_set_refund_to` in my implementation. Maybe update the comment accordingly.